<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-12-31T15:18:51 IST" name="Suite" finished-at="2023-12-31T15:19:26 IST" duration-ms="34768">
    <groups>
      <group name="purchaseorder">
        <method signature="SubmitOrderTest.SubmitOrder(java.util.HashMap)[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" name="SubmitOrder" class="shoppingsite.tests.SubmitOrderTest"/>
      </group> <!-- purchaseorder -->
    </groups>
    <test started-at="2023-12-31T15:18:51 IST" name="SubmitOrderTest" finished-at="2023-12-31T15:19:26 IST" duration-ms="34768">
      <class name="shoppingsite.tests.SubmitOrderTest">
        <test-method is-config="true" signature="launchApp()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:18:52 IST" name="launchApp" finished-at="2023-12-31T15:18:55 IST" duration-ms="3589" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="SubmitOrder(java.util.HashMap)[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:18:55 IST" name="SubmitOrder" data-provider="getData" finished-at="2023-12-31T15:19:04 IST" duration-ms="9254" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{productname=ZARA COAT 3, passwd=Nuzvid@123, email=testpractice1@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitOrder -->
        <test-method is-config="true" signature="closeDriver()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:04 IST" name="closeDriver" finished-at="2023-12-31T15:19:05 IST" duration-ms="407" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
        <test-method is-config="true" signature="launchApp()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:05 IST" name="launchApp" finished-at="2023-12-31T15:19:08 IST" duration-ms="3284" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="SubmitOrder(java.util.HashMap)[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:08 IST" name="SubmitOrder" data-provider="getData" finished-at="2023-12-31T15:19:17 IST" duration-ms="9231" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{productname=ADIDAS ORIGINAL, passwd=Nuzvid@123, email=testpractice1@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitOrder -->
        <test-method is-config="true" signature="closeDriver()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:17 IST" name="closeDriver" finished-at="2023-12-31T15:19:18 IST" duration-ms="401" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
        <test-method is-config="true" signature="launchApp()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:18 IST" name="launchApp" finished-at="2023-12-31T15:19:21 IST" duration-ms="3330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="OrderHistoryTest(java.util.HashMap)[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:21 IST" name="OrderHistoryTest" data-provider="getData" finished-at="2023-12-31T15:19:22 IST" depends-on-methods="shoppingsite.tests.SubmitOrderTest.SubmitOrder" duration-ms="504" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{productname=ZARA COAT 3, passwd=Nuzvid@123, email=testpractice1@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at shoppingsite.tests.SubmitOrderTest.OrderHistoryTest(SubmitOrderTest.java:75)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="closeDriver()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:22 IST" name="closeDriver" finished-at="2023-12-31T15:19:22 IST" duration-ms="410" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
        <test-method is-config="true" signature="launchApp()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:22 IST" name="launchApp" finished-at="2023-12-31T15:19:25 IST" duration-ms="3159" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="OrderHistoryTest(java.util.HashMap)[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:25 IST" name="OrderHistoryTest" data-provider="getData" finished-at="2023-12-31T15:19:26 IST" depends-on-methods="shoppingsite.tests.SubmitOrderTest.SubmitOrder" duration-ms="504" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{productname=ADIDAS ORIGINAL, passwd=Nuzvid@123, email=testpractice1@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at shoppingsite.tests.SubmitOrderTest.OrderHistoryTest(SubmitOrderTest.java:75)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="closeDriver()[pri:0, instance:shoppingsite.tests.SubmitOrderTest@54422e18]" started-at="2023-12-31T15:19:26 IST" name="closeDriver" finished-at="2023-12-31T15:19:26 IST" duration-ms="398" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
      </class> <!-- shoppingsite.tests.SubmitOrderTest -->
    </test> <!-- SubmitOrderTest -->
  </suite> <!-- Suite -->
</testng-results>
